contents/handbook/small-teams/product-analytics/objectives.mdx

```diff
@@ -1,52 +1,30 @@
-### Objective 1: Nail Data Exploration
+## Q1 2023 Goals
 
-1. Build v0.1 of the data exploration system
-2. Run a design sprint with Cory
-3. Ship v2
+### Objective 1: Ship PostHog 3000 UX. 10 happy ICP beta customers.
 
-#### Why?
+- Why? Subjectively, we believe there are many UI/UX improvements to make it a tool that product engineers love.
 
-Product reason:
+- Key results:
+  - We have shipped the core elements of PostHog 3000 that we believe are most impactful. This can be tabs, dark mode, a new query editor. TBD
+  - (Likely Q2) Ship notebooks - making it seamless to explore product questions in PostHog. Build with 5 happy beta customers
+    - Why? This should enable faster and more structured data exploration. The data exploration work this quarter unlocks this as a key workflow.
 
-- We have a pretty good interface for data *visualization* (trends, funnels, retention, etc), but not so much for data 
-  *exploration* (events, persons). Users have been asking for more.
-- The biggest pain comes from not being able to move between different types of related data. Imagine [this scenario](https://github.com/PostHog/posthog/issues/7963#issuecomment-1010235411) (© @rcmarron)
-> 0) Found an anomaly about feature usage in trends
-> 1) "I wonder which paying organizations have used feature Y"
-> 2) Search for that event/action and change the tab to organizations
-> 3) "Interesting CompanyXYZ is using it, I wonder who"
-> 4) Flip to persons + add an organization filter for CompanyXYZ
-> 5) See the persons
-> 6) "I wonder how/why they're using it"
-> 7) Click recordings + see the recordings
-> It would be pretty clunky/impossible to explore your data in that way with today's interface. 
-- Currently you need to re-create this same filtered query on 3-4 different pages and manually copy over the properties. 
-  It's doable, but not a great experience. 
+### Objective 2: Make PostHog performance frustration free for our 10 largest customers
 
-Code reason:
-- We have a lot of bespoke bridges to show related data in the interface (people on insights, recordings on people).
-- Every new one adds more complexity. 
-- This compounded by ever so slightly different use of filters on different data types.
-- The further we continue down this path, the harder it'll be to have a fully integrated product across all data.
+- Key results:
+  - For top 10 US&EU clients p95 of dashboard load time <5s 
+  - For top 10 US&EU clients p95 of insight load time <5s
+  - INSERT METRIC HERE FOR TIMEOUTS
```

Comment:
```suggestion
```

Let's remove this. As per https://github.com/PostHog/posthog/issues/13156 we don't have a way to measure it yet, and getting it _properly_ measurable is considerable effort.

Rather than make up an arbitrary number, I'm gonna argue that this key result is closely correlated of the other two. There might be product reasons to focus on this, but they're not relevant beyond the context of planning projects.